src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -47,7 +48,27 @@
 }
 
 
-def histogram(polygons):
+def histogramPoll(jobId):
+    host = settings.GEOP['host']
+    port = settings.GEOP['port']
+    path = '/jobs/' + jobId
+    conn = httplib.HTTPConnection("%s:%s" % (host, port))
+
+    # Try for about two minutes ...
```

Comment:
Two minutes seems like a long time to wait, but I'm not sure what the expectations are around this. Assuming we want to move in this direction, @mmcfarland probably knows what time limit to set here.